src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyperEngine.java

```diff
@@ -42,12 +42,17 @@ public final class GnarlyGenotyperEngine {
     private Set<Class<? extends InfoFieldAnnotation>> allASAnnotations;
 
     private final int maxAltAllelesToOutput;
+    private final boolean emitPls;
     private final boolean keepAllSites;
     private final boolean stripASAnnotations;
 
-
     public GnarlyGenotyperEngine(final boolean keepAllSites, final int maxAltAllelesToOutput, final boolean stripASAnnotations) {
+        this(keepAllSites, maxAltAllelesToOutput, true, stripASAnnotations);
```

Comment:
Would be good for @ldgauthier to have a quick look at the `GnarlyGenotyper` changes here.

Reply:
Actually on second look these `GnarlyGenotyper` changes are trivial (just a new `emitPls` flag), and so don't require extra scrutiny